products/models.py

```diff
@@ -0,0 +1,78 @@
+from django.db import models
+
+from users.models import User
+from cores.timestamp import TimeStamp
+
+# Create your models here.
+class Product(TimeStamp):
+    name        = models.CharField(max_length=45)
+    price       = models.CharField(max_length=45)
+    size        = models.CharField(max_length=45)
+    description = models.TextField(max_length=1000)
+    feeling     = models.CharField(max_length=45)
+    category    = models.ForeignKey('Category', on_delete=models.CASCADE)
+    user        = models.ForeignKey(User, on_delete=models.CASCADE)
+    howtouse    = models.JSONField()
+    usage       = models.IntegerField()
+    texture     = models.JSONField()
+    scent       = models.CharField(max_length=45)
+    use_image   = models.JSONField()
```

Comment:
4개의 필드에 들어가는 값들이 한 컬럼에 들어갈 수 있도록 만들어야 한다는 얘기였습니다!

```suggestion
    usage    = models.JSONField()
```
아래 json data가 usage라는 한 컬럼에 들어가야 합니다.
```python
    {
        "usage" : "123123123".,
        "texture" : "",
        "scent" : "",
    }
```